src/main/java/uk/gov/hmcts/divorce/divorcecase/task/CaseTask.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.divorce.divorcecase.task;
+
+import uk.gov.hmcts.ccd.sdk.api.CaseDetails;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.State;
+
+import java.util.function.Function;
+
+import static java.util.function.Function.identity;
+import static java.util.stream.Stream.of;
+
+public interface CaseTask extends Function<CaseDetails<CaseData, State>, CaseDetails<CaseData, State>> {
+
+    @SafeVarargs
+    static CaseTaskRunner caseTasks(final Function<CaseDetails<CaseData, State>, CaseDetails<CaseData, State>>... tasks) {
```

Comment:
This could be inside CaseTaskRunner

Reply:
yeah would be nice I think if we can move it there